WIP-CHANGELOG.md

```diff
@@ -16,6 +16,7 @@
 - indices.query.bool.max_clause_count is now configurable.
 - Patched Falco rules for  `k8s_containers` , `postgres_running_wal_e` & `user_known_contact_k8s_api_server_activities`. Will be removed if upstream Falco Chart accepts these.
 - Added the user-permissions available pre-defined alerting roles for opensearch.
+- Self service: UserAdmin are now allowed to add new user admins. Clusterrole and Clusterrolebinding has been added accordingly.
```

Comment:
Adding people to "user-view" does not really make them admin.
```suggestion
- Self service: User admins are now allowed to add new users to the clusterrole user-view. Clusterrole and Clusterrolebinding has been added accordingly.
```

Reply:
Fixed.